packages/rancher-vsphere-csi/charts/questions.yaml

```diff
@@ -44,7 +44,20 @@ questions:
     description: Enable migration of volumes provisioned by in-tree vSphere provider to CSI (Available for vSphere 7.0 U1+ only)
     type: boolean
     default: false
-    group: Migration
+    group: "Features"
```

Comment:
Was this supposed to be changed?

Reply:
I changed this to features because we initially had the migration option by itself and it made sense to name it Migration, but they added 2 other options and grouped them together as "internal features" so it made sense to me to group them together in the questions.yaml as well.

https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/fa891a5f1a2290824c97be19195c3f3b5cd5bdc7/manifests/v2.2.0/deploy/vsphere-csi-controller-deployment.yaml#L175-L183